metadata-db-app/src/cmr/metadata_db/config.clj

```diff
@@ -101,3 +101,7 @@
                      (ingest-exchange-name)
                      (access-control-exchange-name)
                      (deleted-granule-exchange-name)]))
+
+(defconfig ingest-subscription-enabled
```

Comment:
are there other types of subscriptions? If so, we should renamed this to be more specific

Reply:
right now we only have subscriptions that are driven by search and the results get emailed out.